### PR TITLE
ASP admin permission

### PR DESCRIPTION
**Pull request by GNaraya4**
ASP admin permission

### CODE REVIEW

File: tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java [hunk 1]
```diff
@@ -1384,31 +1386,34 @@ public Object answer(InvocationOnMock invocation) {
...
 			ReflectionTestUtils.setField(azureServicePrincipalAccountsService, "vaultAuthMethod", "ldap");
 			when(policyUtils.getCurrentPolicies(userDetails.getSelfSupportToken(), userDetails.getUsername(), userDetails))
 					.thenReturn(latestPolicies);
 			when(reqProcessor.process(Mockito.eq("/sdb"), Mockito.any(), Mockito.eq(token))).thenReturn(getMockResponse(HttpStatus.OK, true,
 					"{\"data\":{\"isActivated\":true,\"managedBy\":\"normaluser\",\"name\":\"svc_vault_test5\",\"users\":{\"normaluser\":\"sudo\"}}}"));
 			when(tokenUtils.getSelfServiceToken()).thenReturn(token);
+			String iamMetaDataStr = "{ \"data\": {\"servicePrincipalName\": \"testaccount\", \"servicePrincipalId\": \"c865a078-a5a7-44d8-9e43-0764c545b76d\", \"tenantId\": \"c865a078-a5a7-44d8-9e43-0764c545b76d\",\"servicePrincipalClientId\": \"c865a078-a5a7-44d8-9e43-0764c545b76d\", \"createdAtEpoch\": 1609754282000, \"owner_email\": \"normaluser@testmail.com\", \"application_id\": \"app1\", \"application_name\": \"App1\", \"application_tag\": \"App1\", \"isActivated\": false, \"secret\":[{\"secretKeyId\":\"testaccesskey\", \"expiryDuration\":604800000}]}, \"path\": \"/azuresvcacc/testaccount\"}";
```
Comment (sivanag1974) [inline tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java:1414]: Can you verify that c865a078-a5a7-44d8-9e43-0764c545b76d is not real

File: tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java [hunk 2]
```diff
@@ -3512,8 +3538,11 @@ public void testOnboardAzureServiceAccountFailure() {
...
+//		String iamMetaDataStr = "{ \"data\": {\"servicePrincipalName\": \"svc_cce_usertestrr16\", \"servicePrincipalId\": \"c865a078-a5a7-44d8-9e43-0764c545b76d\", \"tenantId\": \"c865a078-a5a7-44d8-9e43-0764c545b76d\",\"servicePrincipalClientId\": \"c865a078-a5a7-44d8-9e43-0764c545b76d\", \"createdAtEpoch\": 12345L, \"owner_ntid\": \"normaluser\", \"owner_email\": \"normaluser@testmail.com\", \"application_id\": \"app1\", \"application_name\": \"App1\", \"application_tag\": \"App1\", \"isActivated\": false, \"secret\":[{\"secretKeyId\":\"testaccesskey\", \"expiryDuration\":12345L}]}, \"path\": \"azuresvcacc/svc_cce_usertestrr16\"}";
```
Comment (sivanag1974) [inline tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java:3541]: Why do we need commented lines?

**Review by sivanag1974 — DISMISSED** [review]

File: tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java (hunk 1 above)
Comment (GNaraya4) [reply tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java:1414]: it is not real. copied it from the existing code.

File: tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java (hunk 2 above)
Comment (GNaraya4) [reply tvaultapi/src/test/java/com/tmobile/cso/vault/api/service/AzureServicePrincipalAccountsServiceTest.java:3541]: removed

**Review by sivanag1974 — APPROVED** [review]

**Pull request merged**